classes/form/CustomerFormatter.php

```diff
@@ -145,6 +145,11 @@ public function getFormat()
                         'Birthdate', [], 'Customer'
                     )
                 )
+                ->addAvailableValue('placeholder', Tools::getDateFormat())
+                ->addAvailableValue(
+                    'comment',
+                    '('.$this->translator->trans('Ex.', [], 'Shop.Forms.Help').':'.Tools::formatDateStr('31 May 1970').')'
```

Comment:
Can you put the entire string in trans()? It's important for translation (especially for Right-To-Left languages:

```php
$this->translator->trans('(Ex.: %s)', array(Tools::formatDateStr('31 May 1970'), 'Shop.Forms.Help')
```

Also, since we adopted the symfony standards, it's mandatory to use the `array()` notation instead of `[]`. Even if we all prefer the short tag notation 😢